tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_Constraints.cpp

```diff
@@ -1332,7 +1332,7 @@ void test_compute_tags() {
 }
 }  // namespace
 
-// [[TimeOut, 20]
+// [[TimeOut, 30]
```

Comment:
This has (and already had) imbalanced brackets.  Is this timeout actually being applied?

Reply:
Hah, it wasn't! Good catch! Decreased to 10s, corrected brackets